A new readme file with a screenshot

navintc opened the pull request:
A new screenshot added.
Replaced the old readme with a new readme.md file.
Added the screenshot into the readme.md

navintc committed:
Uploaded a screenshot

A screenshot of Puri has been added

navintc committed:
Added the screenshot to Readme

The screenshot has been added into the readme.md

navintc committed:
Screenshot updated

There was an error. The screenshot didnt showup. Therefore edited again.

navintc committed:
Deleted the original Readme

I guess that the screenshot didnt work because of the extension of this file. Im going try again with a .md file

navintc committed:
Uploaded a .md file

A new readme file added

humdingerb commented on README.md:
```diff
@@ -1,10 +1,12 @@
-
-.
\ No newline at end of file
+README
```
Maybe "Puri" is a better heading. Use markdown to format it.

humdingerb replied:
Before the screenshot, add an intro sentence, like:
> Puri is a chess program to play offline or against somebody over the network. It uses the Stockfish chess engine.

Add more info if you find something interesting.

humdingerb commented on README.md:
```diff
@@ -1,10 +1,12 @@
+
+![Puri screenshot](Screenshot.png "Puri")
+
+you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
```
Change to "You need at least gcc4 to compile Puri."
The command is "setarch x86".

humdingerb commented on README.md:
```diff
@@ -1,10 +1,12 @@
+![Puri screenshot](Screenshot.png "Puri")
+
+you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
+You can compile now the program by writing "jam" in the terminal from inside the source-directory.
```
Change to:
"You compile the program by writing "`make`" in Terminal from inside the 'source' directory."

humdingerb commented on README.md:
```diff
@@ -1,10 +1,12 @@
+you need gcc4 to compile it. That means, if you are using a hybrid perhaps you need to run the command "segcc gcc4" first.
+You can compile now the program by writing "jam" in the terminal from inside the source-directory.
+
+You need at least r1ac3 rc   to be able to run the application without major problems.
```
Remove the whole sentence pointing to R1Alpha3. That's many years in the past...

humdingerb commented on README.md:
```diff
@@ -1,10 +1,12 @@
+
+The source code is very dirty. It needs to be cleaned and restructured and some things reimplemented. I did most of the work 
+late at night and under time pressure I decided to implement the solution that was the fastest to implement. I hope this 
+summer i will have some time to clean it
```
Apparently, he didn't have time that summer, many years ago... :)
Let's remove that whole paragraph, too.

scottmc replied:
yup.  it will live on it the commit history, doesn't need to show up in the readme though. ;)

navintc committed:
Readme.md Update

Cleaned the readme as you instructed humdingerb. Thanks for helping out. Is there any more errors?

navintc commented:
I changed the readme.md file. Check it. Tell me if somethings wrong. Thanks.

Commit:
Update README.md

Small text changes

humdingerb commented:
Thanks. I made some more minor text changes before merging.
